src/core_plugins/kibana/server/lib/export/collect_dashboards.js

```diff
@@ -11,8 +11,8 @@ export async function collectDashboards(savedObjectsClient, ids) {
     };
   });
 
-  const docs = await savedObjectsClient.bulkGet(objects);
-  const results = await Promise.all(docs.map(d => collectPanels(savedObjectsClient, d)));
+  const { saved_objects } = await savedObjectsClient.bulkGet(objects);
```

Comment:
This usage is inconsistent with other usage that does `{ saved_objects: savedObjects }`. It makes me wonder if the verbosity is necessary in the other places (at first I thought the linter was making it necessary)?